lib/webpacker/helper.rb

```diff
@@ -33,6 +33,22 @@ def asset_pack_url(name, **options)
     end
   end
 
+  # Creates a image tag that references the named pack file.
+  # This will use asset_pack_path internally, so most of their behaviors will be the same.
+  #
+  # Example:
+  #
+  #   # In development mode with hot module replacement:
+  #   <%= image_pack_tag 'logo.png' %> # => nil
+  #
+  #   # In production mode:
+  #   <%= image_pack_tag 'logo.png' %> # => "http://example.com/packs/logo-1016838bab065ae1e122.png"
+  def image_pack_tag(name, **options)
+    unless Webpacker.dev_server.running? && Webpacker.dev_server.hot_module_replacing?
```

Comment:
@maschwenk image tag doesn't need to respect hot reloading because images aren't hot reloaded.

Reply:
will adjust in PR with specs

Reply:
thanks 👍